src/comment-counter.test.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please it will be more professional if you don't disable this linters, kindly look through [this](https://www.learnhowtoprogram.com/introduction-to-programming-part-time/arrays-and-looping/debugging-in-javascript-using-a-linter) article. 👍 **The same thing is applicable to index.js file (line 24 and 101). Thanks.**

Reply:
Yeah I think it is no longer useful, we'll fix it

Reply:
It is done as instructed